src/OpenTelemetry.Instrumentation.StackExchangeRedis/StackExchangeRedisCallsInstrumentation.cs

```diff
@@ -35,6 +37,10 @@ internal class StackExchangeRedisCallsInstrumentation : IDisposable
         internal const string ActivityName = ActivitySourceName + ".Execute";
         internal static readonly Version Version = typeof(StackExchangeRedisCallsInstrumentation).Assembly.GetName().Version;
         internal static readonly ActivitySource ActivitySource = new ActivitySource(ActivitySourceName, Version.ToString());
+        internal static readonly IEnumerable<KeyValuePair<string, object>> CreationTags = new[]
```

Comment:
This collection is in the main RedisInstrumentation.cs file, as this is where we stored other static variables,
but I believe those were all only used in the class itself. We could alternatively put it in the Converter class where it's used, which is static (Does having it in static save memory? Most people are only instantiating one copy of the RedisInstrumentation.cs class anyway)